src/components/storageunits/EnvironmentRequirementComponent.js

```diff
@@ -17,28 +17,29 @@ export default class EnvironmentRequirementComponent extends Component {
         temperatureTolerance: '',
         relativeHumidity: '',
         relativeHumidityTolerance: '',
-        inertAir: '',
-        inertAirTolerance: '',
-        renhold: '',
-        lightCondition: '',
+        hypoxicAir: '',
+        hypoxicAirTolerance: '',
+        cleaning: '',
+        lightningConditions: '',
         comments: ''
       }
     }
-
+    this.temperatureOnClick = (temperature) => {
```

Comment:
wrong name. this is confusing to me. There is no click happening here. Call it temperatureOnChange or onChangeTemperature